llama_hub/file/pandas_excel/base.py

```diff
@@ -36,11 +36,12 @@ def __init__(
         super().__init__(*args, **kwargs)
         self._pandas_config = pandas_config
         self._concat_rows = concat_rows
-        self._row_joiner = row_joiner
+        self._row_joiner = row_joiner if row_joiner else "\n"
```

Comment:
is this only the case is row_joiner is explicitly specified as None? wondering how this fix helps

Reply:
Don't think I've found anywhere else yet and it seems to get around https://github.com/jerryjliu/llama_index/issues/6203, unless the example's just wrong

Reply:
i think the example's just wrong

Reply:
since this is a minor non-breaking nit, think i'm going to merge and fix later